js/index.js

```diff
@@ -0,0 +1,17 @@
+import Keyboard from './keyboard/keyboard.js';
+
+const div = document.createElement('div');
+div.className = 'container';
+document.body.append(div);
+const textarea = document.createElement('textarea');
+textarea.className = 'container__textarea';
+textarea.name = 'post';
+textarea.maxLength = '5000';
+textarea.cols = '50';
+textarea.rows = '5';
+div.appendChild(textarea);
+textarea.focus();
```

Comment:
Try to use HTML templates like this:
``` javascript
const createdKeyboard = keyboard.draw();

const container = `<div class='container'>
                    <textarea class='container__textarea' name='post' maxlength=5000 cols=50 rows=5 autofocus></textarea>
                    ${createdKeyboard}
                  </div>`;

document.body.innerHTML = container;
```
Then set the template into the DOM at once.
**Try to contact with DOM as rarely as possible. Any operation with DOM is slow.**